packages/react/src/components/Toolbar/Toolbar.tsx

```diff
@@ -98,15 +99,15 @@ class Toolbar extends UIComponent<WithAsProp<ToolbarProps>, ToolbarState> {
       'toggle',
       'custom',
     ]),
-    onReduceItems: PropTypes.func,
-    measurement: customPropTypes.itemShorthand,
-    wrapper: customPropTypes.itemShorthand,
+    overflow: PropTypes.bool,
+    overflowItem: customPropTypes.shorthandAllowingChildren,
   }
 
   static defaultProps = {
     accessibility: toolbarBehavior,
-    measurement: {},
-    wrapper: {},
+    overflowItem: {
+      icon: { name: 'more', outline: true },
```

Comment:
We don't have merging of shorthands, it means that if I will pass:

```tsx
<Toobar overflowItem={{ onClick }} />
```

It will render `ToolbarItem` without any icon 😕 
I propose to move `icon` to `defaultProps` in `.create()` call